namesys/publisher.go

```diff
@@ -79,8 +79,8 @@ func (p *ipnsPublisher) PublishWithEOL(ctx context.Context, k ci.PrivKey, value
 	return PutRecordToRouting(ctx, k, value, seqnum, eol, p.routing, id)
 }
 
-func (p *ipnsPublisher) getPreviousSeqNo(ctx context.Context, ipnskey key.Key) (uint64, error) {
-	prevrec, err := p.ds.Get(ipnskey.DsKey())
+func (p *ipnsPublisher) getPreviousSeqNo(ctx context.Context, ipnskey string) (uint64, error) {
+	prevrec, err := p.ds.Get(dshelp.NewKeyFromBinary(ipnskey))
```

Comment:
What I mean is here's the function that gets the return from `IpnsKeysForID()` and which got the signature change